examples/lines_bars_and_markers/stairs_demo.py

```diff
@@ -0,0 +1,60 @@
+"""
+=============
+Histline Demo
+=============
+
+This example demonstrates the use of `~.matplotlib.pyplot.stairs`
+for histogram and histogram-like data visualization and an associated
+underlying `.StepPatch` artist, which is
+a contrained version of `.PathPatch` specified by its bins and edges.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.patches import StepPatch
+
+np.random.seed(0)
+h, bins = np.histogram(np.random.normal(5, 3, 5000),
+                       bins=np.linspace(0, 10, 20))
+
+fig, axs = plt.subplots(3, 1, figsize=(7, 15))
+axs[0].stairs(h, bins, label='Simple histogram')
+axs[0].stairs(h, bins+5, baseline=50, label='--//-- w/ modified baseline')
+axs[0].stairs(h, bins+10, baseline=None, label='--//-- w/ no edges')
```

Comment:
```suggestion
axs[0].stairs(h, bins+5, baseline=50, label='Modified baseline')
axs[0].stairs(h, bins+10, baseline=None, label='No edges')
```

This is a personal preference but I find this easier to visually parse.